CAP/gap/DerivedMethods.gi

```diff
@@ -3417,3 +3417,125 @@ AddFinalDerivationBundle( "Adding BasisOfExternalHom and CoefficientsOfMorphism
       
   end
 );
+
+##
+AddFinalDerivationBundle( "InitialObject as the empty coproduct",
+                    [ [ Coproduct, 1 ],
+                      [ UniversalMorphismFromCoproductWithGivenCoproduct, 1 ] ],
+                    [ InitialObject,
+                      UniversalMorphismFromInitialObject,
+                      UniversalMorphismFromInitialObjectWithGivenInitialObject,
+                      ZeroObject,
```

Comment:
Why do you exclude the operations belonging to `ZeroObject` here?

Reply:
Because a ZeroObject is in particular an InitialObject, but since I know that you know that, there must be something deeper behind your question :)

Reply:
As discussed in person, `ZeroObject` should not appear here.